### PR TITLE
Dominia Cape Leg Things

### PR DESCRIPTION
**Pull request by Geevies**
Makes the Dominia Cape back thing get hidden by tails / other clothes.

image
image
image

### CODE REVIEW

**Review by alsoandanswer — APPROVED** [review]

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/factions/dominia.dm [hunk 1]
```diff
@@ -49,6 +49,19 @@
 	icon_override = null
 	contained_sprite = TRUE
 
+/obj/item/clothing/accessory/poncho/dominia_cape/get_mob_overlay(var/mob/living/carbon/human/H, var/mob_icon, var/mob_state, var/slot)
+	var/image/I = ..()
+	if(slot == slot_wear_suit_str)
+		var/image/legge = image(mob_icon, null, "lil_leg_thing", H ? H.layer - 0.01 : MOB_LAYER - 0.01)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:55]: human readable variable names, please.

File: code/modules/clothing/factions/dominia.dm [hunk 2]
```diff
@@ -49,6 +49,19 @@
...
+		I.add_overlay(legge)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:56]: ditto.

File: code/modules/clothing/factions/dominia.dm [hunk 3]
```diff
@@ -49,6 +49,19 @@
...
+	return I
+
+/obj/item/clothing/accessory/poncho/dominia_cape/get_accessory_mob_overlay(mob/living/carbon/human/H, force)
+	var/image/base = ..()
+	var/image/legge = image(icon, null, "lil_leg_thing", H ? H.layer - 0.01 : MOB_LAYER - 0.01)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:61]: ditto.

File: code/modules/clothing/factions/dominia.dm [hunk 4]
```diff
@@ -49,6 +49,19 @@
...
+	base.add_overlay(legge)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:62]: ditto.

**Review by KingOfThePing — APPROVED** [review]

File: code/modules/clothing/factions/dominia.dm [hunk 5]
```diff
@@ -49,6 +49,19 @@
 	icon_override = null
 	contained_sprite = TRUE
 
+/obj/item/clothing/accessory/poncho/dominia_cape/get_mob_overlay(var/mob/living/carbon/human/human, var/mob_icon, var/mob_state, var/slot)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:52]:
```suggestion
/obj/item/clothing/accessory/poncho/dominia_cape/get_mob_overlay(var/mob/living/carbon/human/H, var/mob_icon, var/mob_state, var/slot)
```
H is fine.

File: code/modules/clothing/factions/dominia.dm [hunk 6]
```diff
@@ -49,6 +49,19 @@
...
+	var/image/I = ..()
+	if(slot == slot_wear_suit_str)
+		var/image/cape_backing = image(mob_icon, null, "lil_leg_thing", human ? human.layer - 0.01 : MOB_LAYER - 0.01)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:55]:
```suggestion
		var/image/cape_backing = image(mob_icon, null, "lil_leg_thing", H? H.layer - 0.01 : MOB_LAYER - 0.01)
```
H is fine.

Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:55]: what is "lil_leg_thing"?

File: code/modules/clothing/factions/dominia.dm [hunk 7]
```diff
@@ -49,6 +49,19 @@
...
+		I.add_overlay(cape_backing)
+	return I
+
+/obj/item/clothing/accessory/poncho/dominia_cape/get_accessory_mob_overlay(mob/living/carbon/human/human, force)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:59]:
```suggestion
/obj/item/clothing/accessory/poncho/dominia_cape/get_accessory_mob_overlay(mob/living/carbon/human/H, force)
```
H is fine.

File: code/modules/clothing/factions/dominia.dm [hunk 8]
```diff
@@ -49,6 +49,19 @@
...
+	var/image/base = ..()
+	var/image/cape_backing = image(icon, null, "lil_leg_thing", human ? human.layer - 0.01 : MOB_LAYER - 0.01)
```
Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:61]:
```suggestion
	var/image/cape_backing = image(icon, null, "lil_leg_thing", H? H.layer - 0.01 : MOB_LAYER - 0.01)
```
H is fine.

Comment (SleepyGemmy) [inline code/modules/clothing/factions/dominia.dm:61]: what is "lil_leg_thing"?

File: code/modules/clothing/factions/dominia.dm (hunk 5 above)
Comment (NonQueueingMatt) [reply code/modules/clothing/factions/dominia.dm:52]: not really necessary

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
see comments above.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**